src/components/styles/fonts.ts

```diff
@@ -0,0 +1,112 @@
+import { css } from "@emotion/react";
+
+const fontStyles = css`
+  /**
+	* Euclid
+	*/
+
+  /* Semibold */
+  @font-face {
+    font-family: "Euclid Circular A";
+    font-weight: 700;
+    font-style: normal;
+    src: url("/static/fonts/EuclidCircularA/EuclidCircularA-Semibold-WebXL.woff2")
+        format("woff2"),
+      url("/static/fonts/EuclidCircularA/EuclidCircularA-Semibold-WebXL.woff")
+        format("woff");
+  }
+
+  /* Semibold Italic */
+  @font-face {
+    font-family: "Euclid Circular A";
+    font-weight: 700;
+    font-style: italic;
+    src: url("/static/fonts/EuclidCircularA/EuclidCircularA-SemiboldItalic-WebXL.woff2")
+        format("woff2"),
+      url("/static/fonts/EuclidCircularA/EuclidCircularA-SemiboldItalic-WebXL.woff")
+        format("woff");
+  }
+
+  /* Medium */
+  @font-face {
+    font-family: "Euclid Circular A";
+    font-weight: 500;
+    font-style: normal;
+    src: url("/static/fonts/EuclidCircularA/EuclidCircularA-Medium-WebXL.woff2")
+        format("woff2"),
+      url("/static/fonts/EuclidCircularA/EuclidCircularA-Medium-WebXL.woff")
+        format("woff");
+  }
+
+  /* Medium Italic */
+  @font-face {
+    font-family: "Euclid Circular A";
+    font-weight: 500;
+    font-style: italic;
+    src: url("/static/fonts/EuclidCircularA/EuclidCircularA-MediumItalic-WebXL.woff2")
+        format("woff2"),
+      url("/static/fonts/EuclidCircularA/EuclidCircularA-MediumItalic-WebXL.woff")
+        format("woff");
+  }
+
+  /* Normal */
+  @font-face {
+    font-family: "Euclid Circular A";
+    font-weight: 400, normal;
```

Comment:
400 [corresponds to normal](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-weight#common_weight_name_mapping) so this is superfluous (also not positive it's valid CSS)
```suggestion
    font-weight: 400;
```